_posts/0000-01-02-ajay999.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
+Your text
+Use the left attow to go back!
```

Comment:
Here is the another line after the second slide
![p-assorted-10-gerberas-in-basket-arrangement-112354-m](https://user-images.githubusercontent.com/8430884/87525698-7ac83280-c6a7-11ea-9051-1d7b8456f3c5.jpg)